ChangeLog.md

```diff
@@ -6,6 +6,10 @@
   - Making `HyperRectDomain_(sub)Iterator` random-access iterators
     (allowing parallel scans of the domain, Roland Denis,
     [#1416](https://github.com/DGtal-team/DGtal/pull/1416))
```

Comment:
Of the ?

(Sorry I cannot add suggestions on the iOS app)

Reply:
yes ;) thanks